src/server/usage/README.md

```diff
@@ -0,0 +1,92 @@
+# Kibana Telemetry Service
+
+Telemetry allows Kibana features to have usage tracked in the wild. The general term "telemetry" refers to multiple things:
+
+1. Integrating with the telemetry service to express how to collect usage data (Collecting).
+2. Sending a payload of usage data up to Elastic's telemetry cluster, once per browser per day (Sending).
+3. Viewing usage data in the Kibana instance of the telemetry cluster (Viewing).
+
+You, the feature or plugin developer, mainly need to worry about the first meaning: collecting. To integrate with the telemetry services for usage collection of your feature, there are 2 steps:
+
+1. Create a usage collector using a factory function
+2. Register the usage collector with the Telemetry service
+
+NOTE: To a lesser extent, there's also a need to update the telemetry payload of Kibana stats and telemetry cluster field mappings to include your fields. This part is typically handled not by you, the developer, but different maintainers of the telemetry cluster. Usually, this step just means file an issue in the telemetry repo and have someone approve your data model.
```

Comment:
Should we remove the reference to the telemetry repo here, since it's private?

Reply:
Yeah, it isn't great to have this. I'll change it to tell folks to talk to the Platform team.